testing/php7-yaml/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+pkgname=php7-yaml
+_pkgreal=yaml
+pkgver=2.0.2
+pkgrel=0
+pkgdesc="YAML syntax bindings for PHP"
+url="http://bd808.com/pecl-file_formats-yaml/"
+arch="all"
+license="MIT"
+makedepends="php7-dev autoconf yaml-dev"
+depends="php7-common"
+builddir="$srcdir"/$_pkgreal-$pkgver/
+
+# Set source file explicitly, otherwise extracted archive directory breaks the build.
+source="$pkgname-$pkgver.tgz::http://pecl.php.net/get/$_pkgreal-$pkgver.tgz
+10-detect.patch"
+
+# Could not make the test suite pass
+# Nor the 10-detect.patch enough to pass all tests. TODO
+# https://github.com/alpinelinux/aports/pull/1531#issuecomment-340331437
+options="!check"
+
+prepare() {
+	default_prepare
+	patch -p0 "$builddir"detect.c < "$srcdir"/10-detect.patch
```

Comment:
I couldn't make this patch work, nor it seems to be enough to make all test pass.
Maybe you can help here @TBK